Daramola98/WEconnectApp
# #158723511 Redesign Landing Page

#### What does this PR do?
This PR redesigns landing page of the application.

#### Description of Task to be completed?
- Draw mockup for new landing page
- Implement mockup

#### How should this be manually tested?
Setup project following instructions in readme then run the command landing page should be different from the previous one

#### What are the relevant pivotal tracker stories?
#158723511
